datafusion/core/src/execution/context/mod.rs

```diff
@@ -356,6 +357,52 @@ impl SessionContext {
         }
     }
 
+    /// Enable the dynamic file query for the current session.
+    /// See [DynamicFileCatalog] for more details
+    ///
+    /// # Example: query the url table
+    ///
+    /// ```
+    /// # use datafusion::prelude::*;
+    /// # use datafusion::{error::Result, assert_batches_eq};
+    /// # #[tokio::main]
+    /// # async fn main() -> Result<()> {
+    /// let cfg = SessionConfig::new().set_str("datafusion.catalog.has_header", "true");
```

Comment:
I think @korowa  changed the default so that `has_header` is true by default now so this line is unecessary